src/sql/parts/modelComponents/button.component.ts

```diff
@@ -82,6 +82,9 @@ export default class ButtonComponent extends ComponentWithIconBase implements IC
 						args: e
 					});
 				}
+
+				// unfocus the button when it is already clicked
+				this._button.blur();
```

Comment:
Can we test the dialog & wizard scenarios to verify this doesn't negatively impact them?

Reply:
I removed blur(). The correct scenario is to focus on the new UI (ie. focus on new cell, terminal window)

---
In reply to: [216546651](https://github.com/Microsoft/sqlopsstudio/pull/2488#discussion_r216546651) [](ancestors = 216546651)